vllm/sequence.py

```diff
@@ -583,12 +591,21 @@ def num_seqs(self, status: Optional[SequenceStatus] = None) -> int:
         if status is None:
             return len(self.seqs)
 
+        if self.is_single_seq:
```

Comment:
in the future I will try to remove sequencegroup, but gald to see it works here.

Reply:
yeah not sure if beam search has value anymore